src/Calculator/Views/MainPage.xaml

```diff
@@ -151,6 +151,10 @@
             </Button>
         </Grid>
 
+        <Grid Grid.Row="1" Canvas.ZIndex="0">
+            <Frame x:Name="SettingsHolder" Visibility="Collapsed"/>
```

Comment:
This has an odd bug, the page behind the settings page is still loaded so when I use keyboard navigation I can still access UI elements behind the settings page. For example I can pop up the 'scientific' menu from scientific calculator with the keyboard. Can we handle this page like the other calculator modes?

![image](https://user-images.githubusercontent.com/6334170/108274234-76766300-7129-11eb-90fe-7b7f34105a69.png)

Reply:
@joseartrivera I tried to handle it that way, but I can't figure out how to navigate back to the previous page. This is easier since pressing the "Back" button just collapses the frame.

Reply:
@joseartrivera If we don't use the collapse frame method for the back button, what should we do?

Reply:
@Chips1234 we should utilize [page ](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.page?view=winrt-19041) navigation or [frame](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Frame?view=winrt-19041 )navigation instead

Reply:
@joseartrivera Hello, since we are now handling the settings page like a normal calculator page, do you have a idea how we can store the previous page on file, then navigate back to it? (My previous solution, to just collapse the frame containing the page was much easier to program, in my opinion.) Thanks, have a great Easter.

Reply:
Would you be able to use the GoBack() method on the frame? https://docs.microsoft.com/en-us/windows/uwp/design/basics/navigate-between-two-pages

Reply:
@joseartrivera I don't think we can do that, since the other pages are not contained in a frame.

Reply:
@joseartrivera I'll try it though, thank you for your suggestion.

Reply:
I see, how do we handle switching between different modes today? Maybe sticking to that pattern would work.

Reply:
@joseartrivera Hello, I tried using `rootFrame->Navigate(SettingsPage::typeid);` and to go back, `rootFrame->GoBack();`. It works fine, but when I close the application during debugging, it throws an exception saying that "Source not available." If you would like to debug, here's the branch I used to experiment: [https://github.com/Chips1234/calculator/tree/experimental](https://github.com/Chips1234/calculator/tree/experimental). Do you know how to fix that? Thanks.

Reply:
OK, @joseartrivera. The SettingsPage is now navigated via rootFrame, and when the back button is clicked, it navigates back to MainPage. This method should resolve the <key>tab</key> problem. Could you please review this PR again? Thanks, have a great rest of your weekend!

Reply:
Awesome! Thanks for the updates, I will re-review this soon.

Reply:
> 
> 
> Awesome! Thanks for the updates, I will re-review this soon.

Thanks.